app/controllers/error_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ErrorController < ApplicationController
```

Comment:
We have controller tests for the other controllers, so should do the same for this one?